### PR TITLE
Make code black compliant, add positive/negative_friction_tipping_point_nap as property to PileCalculation

### PR DESCRIPTION
**Pull request by jmmaljaars**
Oftentimes one needs to report on the location of the tipping point for the postive/negative friction. Now added as `property` to the `PileCalculation` class and its children.
Re-style code to comply with `black` styling.

### CODE REVIEW

File: tests/test.py [hunk 1]
```diff
@@ -116,6 +116,11 @@ def test_negative_friction(self):
         df = soil.join_cpt_with_classification(self.gef, layer_table)
 
         tipping_point = nap_to_depth(self.gef.zid, -2.1)
+        # print(tipping_point)
```
Comment (ritchie46) [inline tests/test.py]: Comments

File: tests/test.py [hunk 2]
```diff
@@ -116,6 +116,11 @@ def test_negative_friction(self):
...
+        # print(self.gef.zid)
```
Comment (ritchie46) [inline tests/test.py]: Comments. Used for testing?

File: requirements.txt [hunk 3]
```diff
@@ -1,5 +1,6 @@
+git+https://github.com/ritchie46/pygef.git@master
```
Comment (ritchie46) [inline requirements.txt]: Pygef is already in requirements. And I want to youse the pypi versions as they are better tested than the master is.

Comment (jmmaljaars) [reply requirements.txt]: Solved in https://github.com/ritchie46/anaPile/pull/1/commits/91cddcb14a884b5661e3f5680146a7e31d0383c0#diff-b4ef698db8ca845e5845c4618278f29aL1

File: tests/test.py (hunk 2 above)
Comment (jmmaljaars) [reply tests/test.py]: Redundant comments removed in https://github.com/ritchie46/anaPile/pull/1/commits/91cddcb14a884b5661e3f5680146a7e31d0383c0#diff-b131fe2646b3e4980d06f90692dc2401L119

**Pull request merged**